jac-uk/admin
# Feature/2458 Add filters to Gantt chart

## What's included?
Closes #2458 

Related PRs:
- [digital-platform: Feature/admin 2458 Update Firestore Indexes #1152](https://github.com/jac-uk/digital-platform/pull/1152)
- [jac-kit: Feature/Update table filters #212](https://github.com/jac-uk/jac-kit/pull/212)

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

Preview URL:
- [Dev](https://jac-admin-develop--pr2504-feature-2458-add-fil-utv430c8.web.app/exercises/programme-view)
- [Production](https://jac-apply-admin-production--pr2504-feature-2458-add-fi-r6234wm7.web.app/exercises/programme-view)

Steps:
1. Go to Programme View.
2. Check if the filters work correctly.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

[admin#2458 Add filters to Gantt chart.webm](https://github.com/user-attachments/assets/f42d2f09-fbbc-4dbb-bc99-8a76ef3cb920)


## Related permissions
Have permissions been considered for this functionality?
- No permission changes required
- Permissions have been added / updated. Details:

---
PREVIEW:PRODUCTION
_can be OFF, DEVELOP or STAGING_
